acceptance/selftest/diff/test.toml

```diff
@@ -0,0 +1,2 @@
+# diff.py does not support stateful comparison of UUIDs.
+DisableComparableUuidReplacement = true
```

Comment:
Please also modify selftest/basic to output 3 uuids to see this feature in action there.